src/extension.js

```diff
@@ -119,11 +193,11 @@ function convert_types_to_array (types) {
  */
 async function get_all_content (types) {
   types = await Promise.all(types)
-  return types.map(async type => {
-    let content = await fetch_content(type)
+  for (const type of types) {
```

Comment:
Usually, to denote that `type` won't be changed but rather a new value will be inferred from `type`. I'm glad you asked this, however - since further down, I actually do change `type` by setting `type.content`. I will refactor this to use `let` eventually.